src/Sarif.Viewer.VisualStudio.UnitTests/ErrorList/SarifErrorListItemTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif;
+using Xunit;
+
+namespace Microsoft.Sarif.Viewer.VisualStudio.UnitTests
+{
+    public class SarifErrorListItemTests
+    {
+        /// <summary>
+        /// Constructor required to set a static member of the SarifViewerPackage class.
```

Comment:
MikeFan has a pet peeve about doc comments :-) Just make this a regular comment.

Reply:
Also I don't think we need to comment the ctor itself; just the `new` line -- how about this, a single comment on the `new` line:

Creating a SarifViewerPackage object has the side effect of setting a static variable in that class, without which the subsequent tests will fail. In production code, the package object is always created before any SarifErrorListItem objects.

---
In reply to: [135302513](https://github.com/Microsoft/sarif-sdk/pull/595#discussion_r135302513) [](ancestors = 135302513)